README.md

```diff
@@ -203,7 +203,7 @@ var vnode = h('div', {style: {color: '#000'}}, [
 
 ### `snabbdom/tovnode`
 
-Converts a DOM node into a virtual node. Especially good for patching over an pre-existing, 
+Converts a DOM node into a virtual node. Especially good for patching over an pre-existing,
```

Comment:
```suggestion
Converts a DOM node into a virtual node. Especially good for patching over an pre-existing, 
```

Reply:
these whitespaces were removed by my editor automatically. I think this is good to not leave trailing whitespaces

Reply:
It's good. Yet, I am strict about commit scope.